ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/port-channel-interfaces.j2

```diff
@@ -243,6 +243,9 @@
 {%             endif %}
 {%         endfor %}
 {%     endif %}
+{# IP NAT Documentation #}
+{%     set ip_nat_interfaces = port_channel_interfaces %}
```

Comment:
please add an `if` block around this include. Same for the other interface types. Thanks.

Reply:
What should be checked by the `if` statement? Line 2 already verifies that `port_channel_interfaces` is defined. We always need to include `interfaces-ip-nat.j2`, because at this point there is no information available if `ip_nat` is defined in one of the interfaces.

Reply:
Good point. Disregard this request.